src/natcap/invest/ui_server.py

```diff
@@ -15,8 +16,15 @@
 from natcap.invest import usage
 
 LOGGER = logging.getLogger(__name__)
+logging.getLogger('flask_cors').level = logging.DEBUG
 
+PREFIX = 'api'
 app = Flask(__name__)
+CORS(app, resources={
+    f'/{PREFIX}/*': {
+        'origins': 'http://localhost:*'
+    }
```

Comment:
This is allowing us to side step CORS security concerns and access files from the localhost?

Reply:
Yes, it's specifically allowing CORS requests from localhost origins (vite dev server files) to endpoints matching `/{PREFIX}/*`